keps/sig-network/1672-tracking-terminating-endpoints/kep.yaml

```diff
@@ -20,16 +20,18 @@ see-also:
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.22"
+latest-milestone: "v1.24"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.20"
+  beta: "v1.22"
```

Comment:
my apologies -- seems like I never got around to coming back to update this KEP after some of the last minute changes that happened in v1.22